### DISCUSSION TITLE
Is there going to be an update?

### DISCUSSION THREAD

**Discussion by TheepStar** [mRemoteNG/mRemoteNG#2515] [Q&A] [answered]
I was wondering if a new release is on the horizon and if so, would it be resolving the following?

https://nvd.nist.gov/vuln/detail/CVE-2023-30367

**Comment by Kvarkas** [comment, marked as answer]
yes, new approatch - mR dont save passwords in tree anymore, insted save a link to sensitive information source

**Reply by evanhwolf** [reply]
I'm very much looking forward to using this for work, but I'm disallowed due to the outstanding vulnerability.

Any idea when this update should be expected?

**Comment by neilpeel13** [comment]
Any update/news on the project please, guys?

Thanks for everyone's efforts on this! :)

**Comment by Kvarkas** [comment]
hello, yes working tough on mR, I was promice early I know, but job, family... you named, but finaly we have freeze in IT systems that mean relax days at primary job, so could focused on RC :) at the moment finishing coding for settings and recode tree structure so mR will not be responcive any more for any sensitive data storage, that will be fully end user duty, should be manage to release before xmas

**Reply by neilpeel13** [reply]
Superb, thanks for the update - and hopefully all goes well!  :)

Happy holidays to all!

**Comment by Risae** [comment]
@Kvarkas I didn't see any GitHub Actions pipeline to create nightly releases, are you guys planning to add GitHub Actions to the Repository?

**Comment by Kvarkas** [comment]
we are using [appveyour](https://www.appveyor.com/) integration for such

**Comment by Risae** [comment]
> we are using [appveyour](https://www.appveyor.com/) integration for such

Ah, okay. Could you send me the script that you are using to build mRemoteNG with appveyour so i can try to create a GitHub Actions pipeline?

**Comment by Kvarkas** [comment]
no :) I mean I am not sure how to do it, as we have integration, once we do commit appveyor try to build it and do the rest, so basicaly its controled by it, its couple of processes what a sit on it so its not strait forward and some actions are using encryption and passwords what i prefer to keep away from GitHub :) (keep in mind that co-pilit, chatgpt and bunch of others)

**Comment by Risae** [comment]
> no :) I mean I am not sure how to do it, as we have integration, once we do commit appveyor try to build it and do the rest, so basicaly its controled by it, its couple of processes what a sit on it so its not strait forward and some actions are using encryption and passwords what i prefer to keep away from GitHub :) (keep in mind that co-pilit, chatgpt and bunch of others)

Got it, but you have a script which tells appveyor to build the binaries, right? Could you provide me that (without secrets), so i can try to replicate the build steps in GitHub Actions?

**Comment by Kvarkas** [comment]
its not like script its an watcher its once got commit it start build in technics to build it, you could register with tem and send me yours nick in private, will add you into our project so you could see how its made

**Comment by neilpeel13** [comment]
I guess the Christmas 'possible' release encountered some roadblocks.

Any news guys? How are things progressing?

**Comment by georgi-sotirov** [comment]
hello, 

Any news regarding the update? Do you guys plan it soon? I know near Christmas time was the plan, but do you guys have some progress?

best wishes,
Georgi

**Comment by Kvarkas** [comment]
hey there, yes I am progressing but due a lot of changes in all around and keep in mind i am working allone having wife and kids :) as well as primary job so its take time... but its not abandoned but i dont like release half ready product so be patient :) So far sorted issues with putty / puttyng, just sorting certification process and will start releasing it for updates. On mR a huge work is happening regarding improve settings saving and connection (we have vuln registered what is still not sorted due need to change completely a way how we store connection details (at the moment its will be user duty to store it, not our!) :) so wheels are keep moving, sorry if not much visible, all boiling under hood at the moment
